packages/cli/internal/pkg/cli/workflow/workflow_tasks.go

```diff
@@ -32,6 +38,23 @@ func (m *Manager) GetWorkflowTasks(runId string) ([]Task, error) {
 
 }
 
+func (m *Manager) GetRunLog(runId string) (RunLog, error) {
+
+	tasks, err := m.GetWorkflowTasks(runId)
+	if err != nil {
+		return RunLog{}, err
```

Comment:
manager's `err` has to be set to this error value. Please see other method of this manager for an example of this design pattern. 

The whole implementation doesn't feel like it belongs here. Please consider different option.

Reply:
> The whole implementation doesn't feel like it belongs here

Can you elaborate? The other calls here are fetching and providing data from the same source (`WES.getRunLog`), for instance `manager.GetWorkflowTasks` provides a subset of RunLog data.

Reply:
So a manager is used, when one have to execute multiple related commands and handle errors behind the scene. This method/use case doesn't fit this pattern. Our managers are already overly complicated and require some refactoring. So I would advise against putting random methods to it. By random, I mean a method doesn't follow the pattern, doesn't execute multiple commands that can return errors.